CHANGELOG.md

```diff
@@ -14,6 +14,8 @@
 ### Features
 
 - more Sns related features: `notifyParticipation` and `getUserCommitment`
+- new library `@dfinity/utils`
```

Comment:
will do but the utils is not yet integrated / used

Reply:
Probably added during the rebase. I'll remove it.

Reply:
done